.nyc_output/76c64d62-a9ed-41c2-80f1-279e09c5da1d.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
@youngestdj  shouldn't this be git ignored?

Reply:
I have added it to `.gitignore`